bundles/core/src/main/java/com/adobe/cq/commerce/core/components/services/ComponentsConfiguration.java

```diff
@@ -0,0 +1,63 @@
+/*
+ *  Copyright 2020 Adobe. All rights reserved.
```

Comment:
Sorry to be picky, this header is not formatted like all the other headers we use.